.github/workflows/pylint.yml

```diff
@@ -0,0 +1,30 @@
+name: Pylint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Why run on all 3?

Reply:
I don't actually know - that was just the default on the workflow template - no bad thing I suppose